app/assets/javascripts/component_guide/application.js

```diff
@@ -1,2 +1,3 @@
+//= require_tree ../govuk_publishing_components/vendor/polyfills
```

Comment:
Instead of vendoring (and maintaining a polyfill) we could use [the one from govuk-frontend](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/vendor/polyfills/Element/prototype/closest.js). I did [a similar thing](https://github.com/alphagov/govuk_publishing_components/pull/1620/files#diff-835c4c4c8070c469c6e70cfe59357344R2) when re-working the checkboxes.js script.

```suggestion
//= require govuk/vendor/polyfills/Element/prototype/closest.js
```

Reply:
Cool, thanks for that. `govuk-frontend` doesn't seem to have a polyfill for `indexOf`, so I think I'll still need to include that.

Reply:
Updated 👍